client/src/components/CardItem/CardItem.js

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+// import PropTypes from 'prop-types';
+import {Typography, Box, styled} from "@mui/material";
+import ArrowForwardIosIcon from '@mui/icons-material/ArrowForwardIos';
+
+const CardItemWrapper = styled("div")({
+    minWidth: "265px",
+    maxWidth: "265px",
+    height: "400px",
+    background: `url('../../assets/images/hotDeals/aerial-view-of-colorful-mixed-forest-shrouded-in-morning-fog-on-a-beautiful-autumn-day.jpg')`,
+    backgroundSize: "cover",
+    color: "white",
+    position: "relative",
+    cursor: "pointer",
+
+    "&::after": {
+        content: "''",
+        position: "absolute",
+        width: "100%",
+        height: "100%",
+        background: "rgba(0,0,0,0.3)",
+        left: "0",
+        top: "0",
+        transition: "opacity 0.5s",
+        opacity: "1",
+    },
+
+    "&:hover::after": {
+        opacity: "0",
+        cursor: "pointer",
+    }
+})
+
+const CardItemPrice = styled(Typography)(({theme}) => ({
+    backgroundColor: theme.palette.primary.main,
+    textAlign: "center",
+    width: "108px",
+    fontSize: "18px",
+    paddingTop: "19px",
+    paddingBottom: "19px",
+    fontWeight: "bold",
+    letterSpacing: "1px",
+    marginTop: "0",
+    position: "absolute",
+    zIndex: "1",
+    color: theme.palette.common.white,
```

Comment:
please stick to our theme, use h3 for example from typography

Reply:
![image](https://user-images.githubusercontent.com/101405506/200168476-9ee49477-2a18-4efa-b791-216db260ed0c.png)

i don't understand how to fix this when i try to use h3 typography

Reply:
just add this line on the top of CardItem.js file 

![image](https://user-images.githubusercontent.com/107735322/200298390-f83decac-af68-4623-8eb6-dcd56dbe35c1.png)


@Vladislav2498